web3/index.js

```diff
@@ -89,3 +95,17 @@ function loadScripts () {
   });
 }
 
+function configureApi (config) {
+  const { DAPPS } = config;
+
+  const dappsInterface = DAPPS.split(':')[0];
+  const dappsPort = DAPPS.split(':')[1];
+
+  if (dappsInterface) {
+    window.secureApi._dappsInterface = dappsInterface;
```

Comment:
Yes, it is hacky :/ Would be better to expose those from the API, but this means that users must update to a new version of the UI. Will implement it though